### ISSUE TITLE
AttackProtector: detect nick spammers and punish them

### ISSUE THREAD

**Issue by Mikaela** [ProgVal/Supybot-plugins#292]
For example, if line has 5 or more nicks, kban.

**Comment by Mikaela** [comment]
This has recently became a phenomeneon again.

**Comment by sorki** [comment]
sadly banning will get your bot banned :)

**Comment by ProgVal** [comment]
@sorki what?

**Comment by sorki** [comment]
Yes, with the amount of spam that's happening on freenode your bot will just ban people too fast (or too many) and get banned due to it. It's better to switch channel to moderated and give people +v according to some criterion (time, captcha?).
